packages/cardano-services/src/NetworkInfo/NetworkInfoHttpService.ts

```diff
@@ -58,6 +58,13 @@ export class NetworkInfoHttpService extends HttpService {
         logger
       )
     );
+    router.post(
+      '/current-wallet-protocol-parameters',
+      providerHandler(networkInfoProvider.currentWalletProtocolParameters.bind(networkInfoProvider))(
+        HttpService.routeHandler(logger),
+        logger
+      )
+    );
```

Comment:
Don't quite understand why you needed to change so many files (provider interface etc.).

Can't it just `networkInfoProvider.protocolParameters.bind(networkInfoProvider)`?

Reply:
Seems we can attach it directly here. I had a mixed local changes state and was misled that we need to apply the type changes, thanks for pointing that out ! @mkazlauskas